_sass/pages/_what_is_login.scss

```diff
@@ -0,0 +1,48 @@
+.container.what-is-login {
+  .one-account {
+    background: url(../img/what-is-login/one-account/what-is-one-account.svg) left center no-repeat;
```

Comment:
I noticed that this file is 4.45MB (3.1MB after GZIP). That seems very very large. Is there any way to shrink it? It's noticeably slow to load even on a good connection.

Reply:
Agreed.  I'll cut a PR to use the PNGs instead.